admin/views/tool-bulk-editor.php

```diff
@@ -61,7 +61,7 @@ function get_rendered_tab( $table, $id ) {
 
 ?>
 <script>
-	var wpseo_bulk_editor_nonce = '<?php echo wp_create_nonce( 'wpseo-bulk-editor' ); ?>';
+	var wpseoBulkEditorNonce = '<?php echo wp_create_nonce( 'wpseo-bulk-editor' ); ?>';
```

Comment:
For backwards compatibility we stil want to expose the variable as it was used before, please add a reference for it:

```js
//eslint-ignore-next-line
var wpseo_bulk_editor_nonce = wpseoBulkEditorNonce;
```